client/boot/app.js

```diff
@@ -0,0 +1,56 @@
+// Initialize localStorage polyfill before any dependencies are loaded
+require( 'lib/local-storage' )();
+if ( process.env.NODE_ENV === 'development' ) {
+	require( 'lib/wrap-es6-functions' )();
+}
+
+/**
+ * External dependencies
+ */
+import debugFactory from 'debug';
+import { invoke } from 'lodash';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+import {
+	configureReduxStore,
+	locales,
+	setupMiddlewares,
+	utils
+} from './common';
+import createReduxStoreFromPersistedInitialState from 'state/initial-state';
+import detectHistoryNavigation from 'lib/detect-history-navigation';
+import userFactory from 'lib/user';
+
+const debug = debugFactory( 'calypso' );
+
+const boot = currentUser => {
+	debug( "Starting Calypso. Let's do this." );
+
+	const project = require( `./project/${ config( 'project' ) }` );
```

Comment:
>  suspect that this will cause all of ./project/*.js to be bundled together

It looks that way at a quick glance - I can take a closer look shortly to confirm :)